src/backend/models/enums.py

```diff
@@ -16,8 +16,23 @@
 #     along with FMTM.  If not, see <https:#www.gnu.org/licenses/>.
 #
 
-from enum import Enum, IntEnum
-
+from enum import Enum, IntEnum, StrEnum
+
+
+class category(StrEnum, Enum):
```

Comment:
Should these specific categories eventually live in some kind of external file rather than within a class definition? I'm particularly thinking of users who aren't immersed in the standard OSM world (scientists come to mind), who may be looking for data that don't fall within these categories.

Reply:
This seems to be how FastAPI works, I just did what already existed. This just adds the ones for xlsforms.